packages/tools/devtools/devtools-browser-extension/src/popup/PopupScript.ts

```diff
@@ -19,7 +23,26 @@ import { browser } from "../Globals";
 //   If not, we may want to display an error message with a link to docs explaining how to
 //   use them.
 
-browser.tabs.query({ active: true, currentWindow: true }, (tab) => {
+// TODO: double check that tab selection can't change while popup is being displayed.
```

Comment:
We should verify this before merging. If it is possible for the tab selection to change while the popup is up, we should file a follow-up work item to handle tab selection changes gracefully, and update the TODO comment. If it isn't possible, we can just remove this comment :)

Reply:
@WayneFerrao I take it that it's not possible for the tab selection to change while the popup is being displayed? If so, let's leave a note to that effect as a comment, in case a future reader is wondering about this.